evm.md

```diff
@@ -1284,7 +1290,7 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          </k>
 
     rule <k> #mkCall ACCTFROM ACCTTO ACCTCODE BYTES APPVALUE ARGS STATIC:Bool
-          => #touchAccounts ACCTFROM ACCTTO ~> #loadProgram BYTES ~> #initVM ~> #precompiled?(ACCTCODE, SCHED) ~> #execute
+          => #loadProgram BYTES ~> #initVM ~> #precompiled?(ACCTCODE, SCHED) ~> #execute
```

Comment:
Why aren't we touching the accounts here? Will they already be touched somewhere else?

Reply:
They are already being touched in the `loadTx` rule from `driver.md`.

Reply:
Yeah, but there are many ways to call `#mkCall`, and `loadTx` is not always called by it.

`#mkCall` is called for every `CALL*` instruction. Also, the EIP 2929 specifically says that when calls happen, the accounts are added to `<touchedAccounts>`, so I think you should leave it here.